.gitignore

```diff
@@ -24,8 +25,9 @@ monkeytype
 simulations/tickets
 simulations/validation
 simulations/regression_tests/poc
+simulations/regression_tests/poc_configs
 
 */poc
 
-notes.py
+notes.*
```

Comment:
What is this for?

Reply:
This looks like the same thing I use `scratchpad*` for :D Random notes and snippets of code.